www/src/pages/testing.js

```diff
@@ -0,0 +1,68 @@
+import { graphql } from 'gatsby';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import Layout from '../components/Layout';
+import Example from '../components/Example';
+
+const propTypes = {
+  location: PropTypes.object.isRequired,
+  data: PropTypes.shape({
```

Comment:
why do we need all this 🤨

Reply:
we don't i just copy/pasted from the other page